contrib/mesos/pkg/scheduler/executorinfo/registry.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package executorinfo
+
+import (
+	"sync"
+
+	"fmt"
+	"strings"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/mesos/mesos-go/mesosproto"
+	"k8s.io/kubernetes/contrib/mesos/pkg/node"
+	"k8s.io/kubernetes/contrib/mesos/pkg/scheduler/meta"
+	client "k8s.io/kubernetes/pkg/client/unversioned"
+)
+
+type Registry interface {
```

Comment:
do we need to be able to delete entries from this registry to avoid endlessly consuming memory for long running clusters? perhaps upon `Framework.ExecutorLost`?